eng/targets/DoNotDeployToSymStore_Workaround.targets

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  <!-- There is no way to disable this expensive target other than to override it -->
+  <Target Name="DeployToSymStore"
```

Comment:
do we need to do anything to verify that the official build still publishes symbols after this change?

Reply:
the build fails on my machine if `/p:UsingToolPdbConverter=false` is set with `-publish`

Reply:
I'm not sure I understand the answer. Is this a long way to say *no, we don't need to do anything here* :)